tests/providers/common/compat/test_empty.py

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+
+def test_empty():
+    assert True
```

Comment:
ROFL    Were the static checks requiring at least one test?

Reply:
Yeah. The back-compatibility tests were essentially hanging) and I think that missing tests were the reason. We will add likely some real tests soon enough so we will be able to remove that one :).

Reply:
Interesting.  I approved it earlier, before I saw the static check finish (and fail).  Once the static checks are passing, feel free to resolve this.

Reply:
Also, ironic that the back-compatibility tests blocked the back-compatibility provider LOL

Reply:
🤣

Reply:
Well. Initially I copied common.io and had >= 2.8.0 for airflow. so... that was really compat provider trolling itself, because it could not run for Airflow 2.7.0 :D